05_stringer/shahriys/main_test.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+func TestMain(t *testing.T) {
+	var bufstringer bytes.Buffer
+	outstringer = &bufstringer
```

Comment:
see above comment about variable names & camelCase. relevant for bufstringer and outstringer